pydevd_file_utils.py

```diff
@@ -963,3 +963,15 @@ def get_package_dir(mod_name):
         if os.path.isdir(mod_path):
             return mod_path
     return None
+
+def contains_dir(path, dir_name):
```

Comment:
Is this used anywhere? Should it be removed?

Reply:
No it can be removed.